qa/automation/CareerPortal.Tests.Scenarios/1.1 Filters of positions on Main Page.feature

```diff
@@ -0,0 +1,92 @@
+Feature: 1.1 Filters of positions on Main Page
```

Comment:
is it necessary to indent the keywords Given, When, etc.? 
(as in this example https://github.com/shepelevaolga/career_portal_bdd/blob/main/qa/automation/CareerPortal.Tests.Scenarios/2.%20Extended%20example/Example.feature)
I really don't know)
But if necessary, then this is a mistake....(